tensorflow/python/ops/check_ops.py

```diff
@@ -87,6 +87,260 @@ def _shape_and_dtype_str(tensor):
   """Returns a string containing tensor's shape and dtype."""
   return 'shape=%s dtype=%s' % (tensor.shape, tensor.dtype.name)
 
+def _unary_assert_doc(sym, sym_name):
+  """
+  Common docstring for assert_* ops that evaluate a unary predicate over every 
+  element of a tensor.
+
```

Comment:
Lint Error : Missing Returns section in docstring, or wrong choice of Returns/Yields

Reply:
Added Returns section.